lib/jekyll-admin/directory.rb

```diff
@@ -0,0 +1,58 @@
+module JekyllAdmin
+  class Directory
+    extend Forwardable
+    def_delegator :@path, :basename, :name
+    def_delegator :@path, :mtime, :modified_time
+    attr_reader :path, :splat, :content_type, :base
+
+    include Enumerable
+    include JekyllAdmin::URLable
+    include JekyllAdmin::APIable
+
+    TYPE = "directory".freeze
```

Comment:
or directly;

```ruby
{
        :name          => name,
        :modified_time => modified_time,
        :path          => relative_path,
        :type          => "directory",
      }
```

since it is used in only one place.